analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/PublicMethodArgumentsShouldBeCheckedForNull.cs

```diff
@@ -208,7 +208,7 @@ public void Foo(params string[] infixes)
         }
         else
         {
-            RefMethod(ref infixes, infixes.Length); // Noncompliant - FP: infixes is not null at this point
+            RefMethod(ref infixes, infixes.Length); // Noncompliant - FP
```

Comment:
`RefMethod(infixes.Length, ref infixes);` would be a TP instead right? 
If not documented elsewhere, you may want to add the scenario, for future reference and to give insight on what is the underlying issue.

Reply:
That's a FP: in the if statement we first check for null (and even emptiness), so it's safe to use the parameter in the else block.